.github/scripts/.bash_history

```diff
@@ -347,7 +347,7 @@ rm -rf jdk-18_linux-x64_bin.deb
 git rebase -i main
 git rebase -i master
 git stash
-export tempPassword="4QMUMCz8BUpSizlIfnb0XPpCDGir1NyfIxkrkN3Emdc="
+export tempPassword="Placeholder Password, find the real one in the history of the container"
```

Comment:
Not sure how this got changed to be honest, @commjoen is this correct?

Reply:
Will have to check. Hope to do that soon